src/ts/components/playbacktimelabel.ts

```diff
@@ -132,7 +134,15 @@ export class PlaybackTimeLabel extends Label<PlaybackTimeLabelConfig> {
       }
     };
 
+    let playbackReadyHandler = () => {
+      // Set time format depending on source duration
+      this.timeFormat = Math.abs(player.isLive() ? player.getMaxTimeShift() : player.getDuration()) >= 3600 ?
+      StringUtils.FORMAT_HHMMSS : StringUtils.FORMAT_MMSS;
```

Comment:
Could we extract this into a `updateTimeFormatBasedOnDuration` function and maybe also call it from the `playbackTimeHandler`? 🤔 Then we would be fully safe to always show the correct format (especially as I don't see a `DurationChanged` handler)